### ISSUE TITLE
Fix the header for single species results so as not to display spp#

### ISSUE THREAD

**Issue by michaelpirrello** [synrg/dronefly#135]
If the count will include ssp, then spp is not an accurate header.

**Comment by synrg** [comment]
This one is problematic because it's in an area in the code that needs a rewrite. That's causing pain for other tab features we want to roll out, so the rewrite should be done first. I'll file a separate issue about that.
